tests/integration/targets/rds_instance_snapshot/defaults/main.yml

```
@@ -0,0 +1,7 @@
+---
+instance_id: "ansible-test-{{ tiny_prefix }}"
+snapshot_id: "{{ instance_id }}-ss"
+username: test
+password: test12345678
```

for security purpose, we should use a random generated string.

```suggestion
password: "{{ lookup('password', '/dev/null') }}"
```

---

done!